awx/api/serializers.py

```diff
@@ -113,7 +113,7 @@
                                        'total_groups',
                                        'groups_with_active_failures',
                                        'has_inventory_sources'),
-    'project': DEFAULT_SUMMARY_FIELDS + ('status', 'scm_type'),
+    'project': DEFAULT_SUMMARY_FIELDS + ('status', 'scm_type', 'allow_override'),
```

Comment:
this is cool, and smarter than the way I did it in the legacy ui (calling the project detail endpoint to get the info when project changes).  fantastic stuff!

Reply:
Since `scm_type` is already a property of the Project subclass, there should be no additional API database queries from adding this field.

👍

Reply:
Per offline convo with @matburt I am going to undo this and do a different route that does not require changes to to api.